client/src/docs-ui/choose-anchor/choose-anchor.tsx

```diff
@@ -30,7 +30,14 @@ export class DocsChooseAnchor {
             Object.entries(filterMetadataByOptionByName[filterKey]).map(
               ([filterValue, {label, graphicURI}]) => {
                 const route =
-                  this.page && `${this.page.route}?${filterKey}=${filterValue}`;
+                  this.page?.route &&
+                  filterValue === "js" &&
+                  this.page.route.startsWith("/sdk")
```

Comment:
Let's break these out to their own functions

Reply:
Done